otx/algorithms/classification/adapters/mmcls/models/backbone/efficientnetv2.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2018-2022 Kaiyang Zhou
+# SPDX-License-Identifier: MIT
+#
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+"""
+    EfficientNet for ImageNet-1K, implemented in PyTorch.
+    Original papers:
+    - 'EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks,' https://arxiv.org/abs/1905.11946,
+    - 'Adversarial Examples Improve Image Recognition,' https://arxiv.org/abs/1911.09665.
+"""
+
+import os
+
+import timm
+import torch.nn as nn
+from mmcls.models.builder import BACKBONES
+from mmcv.runner import load_checkpoint
+from mpa.utils.logger import get_logger
+
+logger = get_logger()
+
+pretrained_root = "https://github.com/rwightman/pytorch-image-models/releases/download/v0.1-effv2-weights/"
+pretrained_urls = {
+    "efficientnetv2_s_21k": pretrained_root + "tf_efficientnetv2_s_21k-6337ad01.pth",
+    "efficientnetv2_s_1k": pretrained_root + "tf_efficientnetv2_s_21ft1k-d7dafa41.pth",
+}
+
+NAME_DICT = {
+    "mobilenetv3_large_21k": "mobilenetv3_large_100_miil_in21k",
+    "mobilenetv3_large_1k": "mobilenetv3_large_100_miil",
+    "tresnet": "tresnet_m",
+    "efficientnetv2_s_21k": "tf_efficientnetv2_s_in21k",
+    "efficientnetv2_s_1k": "tf_efficientnetv2_s_in21ft1k",
+    "efficientnetv2_m_21k": "tf_efficientnetv2_m_in21k",
+    "efficientnetv2_m_1k": "tf_efficientnetv2_m_in21ft1k",
+    "efficientnetv2_b0": "tf_efficientnetv2_b0",
+}
+
+
+class TimmModelsWrapper(nn.Module):
```

Comment:
TimmModelsWrapper is already declared in DOR. Can this class be removed? If you think that DOR will be removed in the future, you can ignore it.

Reply:
Yeah it will be removed soon. I will leave as is.